desktop/linux/space.md

```diff
@@ -0,0 +1,100 @@
+---
+description: Disk utilization
+keywords: linux, disk
+title: Disk utilization in Docker for Linux
+redirect_from:
+- /linux/space/
+---
+
+Docker Desktop stores Linux containers and images in a single, large "disk image" file in the Linux filesystem. This is different from Docker on Linux, which usually stores containers and images in the `/var/lib/docker` directory.
+
+## Where is the disk image file?
+
+To locate the disk image file, select the Docker icon and then
+**Settings** > **Resources** > **Advanced**.
+
+![Disk preferences](images/menu/prefs-advanced.png){:width="750px"}
+
+The **Advanced** tab displays the location of the disk image. It also displays the maximum size of the disk image and the actual space the disk image is consuming. Note that other tools might display space usage of the file in terms of the maximum file size, and not the actual file size.
+
+## If the file is too big
+
+If the disk image file is too big, you can:
+
+- move it to a bigger drive,
+- delete unnecessary containers and images, or
+- reduce the maximum allowable size of the file.
+
+### Move the file to a bigger drive
+
+To move the disk image file to a different location:
+
+1. Select **Settings** > **Resources** > **Advanced**.
+
+2. In the **Disk image location** section, click **Browse** and choose a new location for the disk image.
+
+3. Click **Apply & Restart** for the changes to take effect.
+
+Do not move the file directly in Finder as this can cause Docker Desktop to lose track of the file.
+
+### Delete unnecessary containers and images
+
+Check whether you have any unnecessary containers and images. If your client and daemon API are running version 1.25 or later (use the `docker version` command on the client to check your client and daemon API versions), you can see the detailed space usage information by running:
+
+```console
+$ docker system df -v
+```
+
+Alternatively, to list images, run:
+
+```console
+$ docker image ls
+```
+
+and then, to list containers, run:
+
+```console
+$ docker container ls -a
+```
+
+If there are lots of redundant objects, run the command:
+
+```console
+$ docker system prune
+```
+
+This command removes all stopped containers, unused networks, dangling images, and build cache.
+
+It might take a few minutes to reclaim space on the host depending on the format of the disk image file:
+
+- If the file is named `Docker.raw`: space on the host should be reclaimed within a few seconds.
+- If the file is named `Docker.qcow2`: space will be freed by a background process after a few minutes.
```

Comment:
Does Docker Desktop for Linux also use both of these file formats? Does this apply there?